staramr/subcommand/Search.py

```diff
@@ -313,6 +317,15 @@ def run(self, args):
         else:
             logger.info("No --pointfinder-organism specified. Will not search the PointFinder databases")
             pointfinder_database = None
+        
+        if(args.plasmidfinder_bacteria):
+            if args.plasmidfinder_bacteria not in PlasmidfinderBlastDatabase.get_available_bacteria():
+                raise CommandParseException("The only Plasmidfinder bacteria(s) currently supported are " + str(
+                    PlasmidfinderBlastDatabase.get_available_bacteria()), self._root_arg_parser)
+            plasmidfinder_database = database_repos.build_blast_database('plasmidfinder', {'bacteria': args.plasmidfinder_bacteria})
+        else:
+            logger.info("No --plasmidfinder-database-type specified. Will search the entire PlasmidFinder database")
```

Comment:
Thanks for adding a message in case `--plasmidfinder-database-type` is not set :+1:

Reply:
No problem! :smile: